include/deal.II/fe/fe_system.h

```diff
@@ -1068,11 +1066,6 @@ class FESystem : public FiniteElement<dim,spacedim>
     ~InternalData();
 
     /**
-     * Flag indicating whether second derivatives shall be computed.
-     */
-    const bool compute_hessians;
```

Comment:
Thinking about it, it's also a bit of an oddity that FESystem should be responsible for computing the second derivatives. Let the base elements do it and then FESystem simply needs to copy the results from the base elements.